modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc

```diff
@@ -7,7 +7,40 @@
 
 *Prerequisites*
 
-* You have first installed the  link:{kn_cli_install_url}[Knative CLI].
+* You have first installed the link:{kn_cli_install_url}[Knative CLI].
+* link:{java_install_url}[Java] {java_min_version} is installed.
+* link:{maven_install_url}[Maven] {maven_min_version} or later is installed.
+* (Optional) link:{docker_install_url}[Docker] is installed.
+* (Optional) link:{podman_install_url}[Podman] is installed.
```

Comment:
Docker and podman are not optional. Either one of them must be installed.

Reply:
Will fix.